### PR TITLE
Create function to remove leading slashes from a path [#16]

### LINKED ISSUE #16: Create function to remove leading slashes from a path
We need a function to remove the leading slash from a path string if it has one.  Follow all project requirements

### PR DESCRIPTION
**Pull request by coco-b0t**
Closes #16

### CODE REVIEW

**Review by skleinjung — CHANGES_REQUESTED** [review]
Let's use arrow functions instead.

**Review by skleinjung — CHANGES_REQUESTED** [review]
Looking at other things still.

**Review by skleinjung — COMMENTED** [review]
Let's use arrow functions.

**Comment by skleinjung** [comment]
Is this  comment?

File: src/utils/remove-leading-slash.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export function removeLeadingSlash(path: string): string {
```
Comment (skleinjung) [inline src/utils/remove-leading-slash.ts:1]: maybe this is a comment?

File: src/utils/remove-leading-slash.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (skleinjung) [inline src/utils/remove-leading-slash.ts:1]: Let's use arrow functions instead.

File: src/utils/remove-leading-slash.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const removeLeadingSlash = (path: string): string => path.replace(/^\//, '');
```
Comment (skleinjung) [inline src/utils/remove-leading-slash.ts:1]: Can we add tests for the removeLeadingSlash function?

**Comment by coco-b0t** [comment]
Deleting progress on issue because: Issue was edited, and I don't know how to reconcile changes with existing work..

Closing.

**Pull request closed without merging**